zuul.d/project-templates.yaml

```diff
@@ -49,6 +49,15 @@
         - ansible-tox-py36
         - ansible-tox-py37
 
+- project-template:
+    name: ansible-navigator-jobs
+    check:
+      jobs:
+        - ansible-tox-linters
+    gate:
+      jobs:
+        - ansible-tox-linters
```

Comment:
if you want to respin this in the morning, using a yaml anchor, that will help avoid typing duplicate jobs into gate.

eg: https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L111

Reply:
Got it. I shall do that.